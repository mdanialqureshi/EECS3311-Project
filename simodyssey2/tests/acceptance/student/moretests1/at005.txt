-- Test in game precondition.
move(N)
move(E)
move(SW)
status
land
liftoff
wormhole
land
pass
abort

test(1,1,1,1,30)
abort
test(1,1,1,1,30)
abort

move(N)
move(E)
move(SW)
status
land
liftoff
wormhole
land
pass
abort

test(1,1,1,1,30)
abort
test(1,1,1,1,30)
abort

test(1,1,1,1,30)
move(N)
move(E)