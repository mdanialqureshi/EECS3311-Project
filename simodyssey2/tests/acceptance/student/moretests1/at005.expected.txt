  state:0.0, ok
  Welcome! Try test(1,1,1,1,30)
->move(N)
  state:0.1, error
  Negative on that request:no mission in progress.
->move(E)
  state:0.2, error
  Negative on that request:no mission in progress.
->move(SW)
  state:0.3, error
  Negative on that request:no mission in progress.
->status
  state:0.4, error
  Negative on that request:no mission in progress.
->land
  state:0.5, error
  Negative on that request:no mission in progress.
->liftoff
  state:0.6, error
  Negative on that request:no mission in progress.
->wormhole
  state:0.7, error
  Negative on that request:no mission in progress.
->land
  state:0.8, error
  Negative on that request:no mission in progress.
->pass
  state:0.9, error
  Negative on that request:no mission in progress.
->abort
  state:0.10, error
  Negative on that request:no mission in progress.
->test(1,1,1,1,30)
  state:1.0, mode:test(1,1,1,1,30), ok
  Movement:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    E---   *---   P---   ----   PP--   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PY--   ----   O---   PW--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   P*--   P---   P*--   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    PW--   ----   *---   ----   PW--   
->abort
  state:1.1, ok
  Mission aborted. Try test(1,1,1,1,30)
->test(1,1,1,1,30)
  state:2.0, mode:test, ok
  Movement:none
  Sectors:
    [1,1]->[0,E],[-9,W],-,-
    [1,2]->[1,P],[2,P],[3,P],[-7,*]
    [1,3]->[4,P],[5,P],-,-
    [1,4]->[6,P],-,-,-
    [1,5]->[-3,Y],-,-,-
    [2,1]->-,-,-,-
    [2,2]->[-8,*],-,-,-
    [2,3]->[7,P],[-6,Y],-,-
    [2,4]->-,-,-,-
    [2,5]->[8,P],-,-,-
    [3,1]->[9,P],-,-,-
    [3,2]->-,-,-,-
    [3,3]->[-1,O],-,-,-
    [3,4]->[10,P],[-10,Y],-,-
    [3,5]->[11,P],-,-,-
    [4,1]->-,-,-,-
    [4,2]->-,-,-,-
    [4,3]->[-11,W],-,-,-
    [4,4]->[12,P],-,-,-
    [4,5]->-,-,-,-
    [5,1]->-,-,-,-
    [5,2]->[13,P],[-4,Y],-,-
    [5,3]->[14,P],-,-,-
    [5,4]->[-5,*],-,-,-
    [5,5]->[15,P],[-2,W],-,-
  Descriptions:
    [-11,W]->
    [-10,Y]->Luminosity:2
    [-9,W]->
    [-8,*]->Luminosity:5
    [-7,*]->Luminosity:5
    [-6,Y]->Luminosity:2
    [-5,*]->Luminosity:5
    [-4,Y]->Luminosity:2
    [-3,Y]->Luminosity:2
    [-2,W]->
    [-1,O]->
    [0,E]->fuel:3/3, life:3/3, landed?:F
    [1,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [2,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [3,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [4,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [5,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [6,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [7,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [8,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [9,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [10,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [11,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [12,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [13,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [14,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [15,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
  Deaths This Turn:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    EW--   PPP*   PP--   P---   Y---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   *---   PY--   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    P---   ----   O---   PY--   P---   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    ----   ----   W---   P---   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    ----   PY--   P---   *---   PW--   
->abort
  state:2.1, ok
  Mission aborted. Try test(1,1,1,1,30)
->move(N)
  state:2.2, error
  Negative on that request:no mission in progress.
->move(E)
  state:2.3, error
  Negative on that request:no mission in progress.
->move(SW)
  state:2.4, error
  Negative on that request:no mission in progress.
->status
  state:2.5, error
  Negative on that request:no mission in progress.
->land
  state:2.6, error
  Negative on that request:no mission in progress.
->liftoff
  state:2.7, error
  Negative on that request:no mission in progress.
->wormhole
  state:2.8, error
  Negative on that request:no mission in progress.
->land
  state:2.9, error
  Negative on that request:no mission in progress.
->pass
  state:2.10, error
  Negative on that request:no mission in progress.
->abort
  state:2.11, error
  Negative on that request:no mission in progress.
->test(1,1,1,1,30)
  state:3.0, mode:test(1,1,1,1,30), ok
  Movement:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    EP--   PY--   P---   P---   W---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    PP--   ----   ----   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    W---   *---   O---   PPPW   PY--   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    P*--   W---   P---   P---   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    ----   ----   P*--   P---   Y---   
->abort
  state:3.1, ok
  Mission aborted. Try test(1,1,1,1,30)
->test(1,1,1,1,30)
  state:4.0, mode:test, ok
  Movement:none
  Sectors:
    [1,1]->[0,E],-,-,-
    [1,2]->[1,P],-,-,-
    [1,3]->[-10,W],-,-,-
    [1,4]->-,-,-,-
    [1,5]->[2,P],-,-,-
    [2,1]->-,-,-,-
    [2,2]->[3,P],[-5,*],-,-
    [2,3]->[4,P],-,-,-
    [2,4]->-,-,-,-
    [2,5]->-,-,-,-
    [3,1]->[5,P],-,-,-
    [3,2]->[6,P],-,-,-
    [3,3]->[-1,O],-,-,-
    [3,4]->-,-,-,-
    [3,5]->[7,P],-,-,-
    [4,1]->[8,P],-,-,-
    [4,2]->[9,P],[-6,Y],-,-
    [4,3]->[-3,*],-,-,-
    [4,4]->[-9,W],-,-,-
    [4,5]->[-8,W],-,-,-
    [5,1]->[-4,*],-,-,-
    [5,2]->[-7,Y],-,-,-
    [5,3]->[10,P],[-11,*],-,-
    [5,4]->[11,P],[-2,*],-,-
    [5,5]->-,-,-,-
  Descriptions:
    [-11,*]->Luminosity:5
    [-10,W]->
    [-9,W]->
    [-8,W]->
    [-7,Y]->Luminosity:2
    [-6,Y]->Luminosity:2
    [-5,*]->Luminosity:5
    [-4,*]->Luminosity:5
    [-3,*]->Luminosity:5
    [-2,*]->Luminosity:5
    [-1,O]->
    [0,E]->fuel:3/3, life:3/3, landed?:F
    [1,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [2,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [3,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [4,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [5,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [6,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [7,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [8,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [9,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [10,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [11,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
  Deaths This Turn:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    E---   P---   W---   ----   P---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   P*--   P---   ----   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    P---   P---   O---   ----   P---   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    P---   PY--   *---   W---   W---   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    *---   Y---   P*--   P*--   ----   
->abort
  state:4.1, ok
  Mission aborted. Try test(1,1,1,1,30)
->test(1,1,1,1,30)
  state:5.0, mode:test, ok
  Movement:none
  Sectors:
    [1,1]->[0,E],[-5,W],-,-
    [1,2]->[-9,*],-,-,-
    [1,3]->[1,P],-,-,-
    [1,4]->[2,P],[-4,*],-,-
    [1,5]->-,-,-,-
    [2,1]->[3,P],[4,P],-,-
    [2,2]->[5,P],[-3,*],-,-
    [2,3]->[-2,W],-,-,-
    [2,4]->-,-,-,-
    [2,5]->[6,P],[7,P],[-10,*],-
    [3,1]->[8,P],[-7,*],-,-
    [3,2]->-,-,-,-
    [3,3]->[-1,O],-,-,-
    [3,4]->-,-,-,-
    [3,5]->-,-,-,-
    [4,1]->-,-,-,-
    [4,2]->-,-,-,-
    [4,3]->[-8,W],-,-,-
    [4,4]->[-6,*],-,-,-
    [4,5]->-,-,-,-
    [5,1]->[9,P],-,-,-
    [5,2]->[10,P],[-11,Y],-,-
    [5,3]->-,-,-,-
    [5,4]->-,-,-,-
    [5,5]->[11,P],-,-,-
  Descriptions:
    [-11,Y]->Luminosity:2
    [-10,*]->Luminosity:5
    [-9,*]->Luminosity:5
    [-8,W]->
    [-7,*]->Luminosity:5
    [-6,*]->Luminosity:5
    [-5,W]->
    [-4,*]->Luminosity:5
    [-3,*]->Luminosity:5
    [-2,W]->
    [-1,O]->
    [0,E]->fuel:3/3, life:3/3, landed?:F
    [1,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [2,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [3,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [4,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [5,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [6,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [7,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [8,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [9,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [10,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [11,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
  Deaths This Turn:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    EW--   *---   P---   P*--   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    PP--   P*--   W---   ----   PP*-   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    P*--   ----   O---   ----   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    ----   ----   W---   *---   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    P---   PY--   ----   ----   P---   
->move(N)
  state:6.0, mode:test, ok
  Movement:
    [0,E]:[1,1,1]->[5,1,2]
    [1,P]:[1,3,1]->[2,4,1]
    [3,P]:[2,1,1]->[1,1,1]
  Sectors:
    [1,1]->[3,P],[-5,W],-,-
    [1,2]->[-9,*],-,-,-
    [1,3]->-,-,-,-
    [1,4]->[2,P],[-4,*],-,-
    [1,5]->-,-,-,-
    [2,1]->-,[4,P],-,-
    [2,2]->[5,P],[-3,*],-,-
    [2,3]->[-2,W],-,-,-
    [2,4]->[1,P],-,-,-
    [2,5]->[6,P],[7,P],[-10,*],-
    [3,1]->[8,P],[-7,*],-,-
    [3,2]->-,-,-,-
    [3,3]->[-1,O],-,-,-
    [3,4]->-,-,-,-
    [3,5]->-,-,-,-
    [4,1]->-,-,-,-
    [4,2]->-,-,-,-
    [4,3]->[-8,W],-,-,-
    [4,4]->[-6,*],-,-,-
    [4,5]->-,-,-,-
    [5,1]->[9,P],[0,E],-,-
    [5,2]->[10,P],[-11,Y],-,-
    [5,3]->-,-,-,-
    [5,4]->-,-,-,-
    [5,5]->[11,P],-,-,-
  Descriptions:
    [-11,Y]->Luminosity:2
    [-10,*]->Luminosity:5
    [-9,*]->Luminosity:5
    [-8,W]->
    [-7,*]->Luminosity:5
    [-6,*]->Luminosity:5
    [-5,W]->
    [-4,*]->Luminosity:5
    [-3,*]->Luminosity:5
    [-2,W]->
    [-1,O]->
    [0,E]->fuel:2/3, life:3/3, landed?:F
    [1,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [2,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [3,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [4,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [5,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [6,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [7,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [8,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [9,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [10,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [11,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
  Deaths This Turn:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    PW--   *---   ----   P*--   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    -P--   P*--   W---   P---   PP*-   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    P*--   ----   O---   ----   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    ----   ----   W---   *---   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    PE--   PY--   ----   ----   P---   
->move(E)
  state:7.0, mode:test, ok
  Movement:
    [0,E]:[5,1,2]->[5,2,3]
    [1,P]:[2,4,1]->[2,5,4]
    [4,P]:[2,1,2]->[1,2,2]
  Sectors:
    [1,1]->[3,P],[-5,W],-,-
    [1,2]->[-9,*],[4,P],-,-
    [1,3]->-,-,-,-
    [1,4]->[2,P],[-4,*],-,-
    [1,5]->-,-,-,-
    [2,1]->-,-,-,-
    [2,2]->[5,P],[-3,*],-,-
    [2,3]->[-2,W],-,-,-
    [2,4]->-,-,-,-
    [2,5]->[6,P],[7,P],[-10,*],[1,P]
    [3,1]->[8,P],[-7,*],-,-
    [3,2]->-,-,-,-
    [3,3]->[-1,O],-,-,-
    [3,4]->-,-,-,-
    [3,5]->-,-,-,-
    [4,1]->-,-,-,-
    [4,2]->-,-,-,-
    [4,3]->[-8,W],-,-,-
    [4,4]->[-6,*],-,-,-
    [4,5]->-,-,-,-
    [5,1]->[9,P],-,-,-
    [5,2]->[10,P],[-11,Y],[0,E],-
    [5,3]->-,-,-,-
    [5,4]->-,-,-,-
    [5,5]->[11,P],-,-,-
  Descriptions:
    [-11,Y]->Luminosity:2
    [-10,*]->Luminosity:5
    [-9,*]->Luminosity:5
    [-8,W]->
    [-7,*]->Luminosity:5
    [-6,*]->Luminosity:5
    [-5,W]->
    [-4,*]->Luminosity:5
    [-3,*]->Luminosity:5
    [-2,W]->
    [-1,O]->
    [0,E]->fuel:3/3, life:3/3, landed?:F
    [1,P]->attached?:T, support_life?:F, visited?:F, turns_left:N/A
    [2,P]->attached?:T, support_life?:F, visited?:F, turns_left:N/A
    [3,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [4,P]->attached?:T, support_life?:F, visited?:F, turns_left:N/A
    [5,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [6,P]->attached?:T, support_life?:F, visited?:F, turns_left:N/A
    [7,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [8,P]->attached?:T, support_life?:F, visited?:F, turns_left:N/A
    [9,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [10,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [11,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
  Deaths This Turn:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    PW--   *P--   ----   P*--   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   P*--   W---   ----   PP*P   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    P*--   ----   O---   ----   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    ----   ----   W---   *---   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    P---   PYE-   ----   ----   P---   
